Sources/Testing/Events/Recorder/Event.HumanReadableOutputRecorder.swift

```diff
@@ -27,6 +27,11 @@ extension Event {
 
       /// The human-readable message.
       var stringValue: String
+
+      /// A concise version of ``stringValue``, if available.
+      ///
+      /// Not all messages include a concise string.
+      var conciseStringValue: String?
```

Comment:
How is this intended to be used?

Reply:
This is a shorter version of the message's string. The longer version is output to `stderr` by default. The shorter version is what gets passed back in the JSON.